test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- This file is not currently being ignored by Git 😄 as we can see it is currently on GitHub. 
- I kindly suggest that you add `test.md` to your `.gitignore` file, afterwards, you can try removing `test.md` from your `feature-branch` branch repo then try to push your local git repo again to see if `test.md` is ignored.